docs/tutorials.rst

```diff
@@ -1,5 +1,160 @@
 Tutorials
 ----------
+This tutorial emphasizes on the abilities and usage of CppInterOp, Let's get
+started! The tutorial demonstrates two examples one in C and python for
+interoperability.
 
+**Note:: This example library shown below is to illustrate the concept on which
+CppInterOp is based on.** 
+
+Python:
+=======
+
+.. code-block:: bash
+
+   libInterop = ctypes.CDLL(libpath, mode = ctypes.RTLD_GLOBAL)
+   _cpp_compile = libInterop.Clang_Parse
+   _cpp_compile.argtypes = [ctypes.c_char_p]
+
+We are using ctypes for inducting our library, *Clang_Parse* which is part
```

Comment:
Where are Clang_Parse and such defined? Should we not show how the header file looks like at least?